install_config/topics/configuring_for_gce_storage.adoc

```diff
@@ -0,0 +1,53 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+[id='configuring-gce-storage_{context}']
+= Configuring {product-title} to use GCP storage
+
+{product-title} can leverage GCP storage using persistent volumes mechanisms.
+{product-title} creates the disk in GCP and attaches the disk to the proper
+instance.
+
+.Procedure
+
+. {product-title} creates the following `storageclass` when you use the `gce-pd`
```

Comment:
Because you maybe configured OCP without GCP ansible variables or you just accidentally deleted the storageclass and you can see the default one here.

Reply:
@bfallonf, I'd include this context. :)